src/shadertools/lib/shader-cache.js

```diff
@@ -1,7 +1,10 @@
 import {VertexShader, FragmentShader} from '../../webgl/shader';
+import Program from '../../webgl/program';
 import assert from 'assert';
 
-const ERR_SOURCE = 'ShaderCache expects source strings';
+const ERR_SOURCE = 'ShaderCache expects source shader strings';
```

Comment:
Do we need these messages? They take up space in our bundle. Maybe just assert without message is enough?

Reply:
Will remove the strings.